sdks/java/io/google-cloud-platform/src/main/java/org/apache/beam/sdk/io/gcp/bigquery/StorageApiWriteUnshardedRecords.java

```diff
@@ -19,12 +19,14 @@
 
 import static org.apache.beam.vendor.guava.v26_0_jre.com.google.common.base.Preconditions.checkArgument;
 
+import com.google.api.core.ApiFuture;
 import com.google.cloud.bigquery.storage.v1.AppendRowsResponse;
 import com.google.cloud.bigquery.storage.v1.ProtoRows;
 import com.google.cloud.bigquery.storage.v1.WriteStream.Type;
 import com.google.protobuf.ByteString;
 import com.google.protobuf.DynamicMessage;
 import java.io.IOException;
+import java.time.Instant;
```

Comment:
any reasons not to use joda time here? totally fine with it, but just wondering

Reply:
java Instant is the recommended class to use. We continue using Joda in public APIs (and in encoded bytes) for backwards compatibility reasons, but for internal uses like this java time is nicer to use.